charts/egeria-presentation/README.md

```diff
@@ -0,0 +1,16 @@
+# egeria-presentation
+
+This repository is based on https://github.com/odpi/egeria-charts, but it only deploys the Egeria UI. This chart assumes that the platform of the view server was deployed using [egeria-platform](../egeria-platform/README.md).
+
+## Deploy Egeria UI
+
+Use the following settings in `egeria-presentation` helm chart:
+- set `egeria.platformConfigMap` to the configmap of the Egeria OMAG deployment (here `egeria-platform-config`)
```

Comment:
worth mentioning it needs to be installed in the same namespace

Reply:
Fixed in bc0532387a29